core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -62,6 +63,12 @@ protected[core] case class Resource(namespace: EntityPath,
   override def toString: String = id
 }
 
+trait EntitlementSpiProvider extends Spi {
+  def entitlementProvider(config: WhiskConfig, loadBalancer: LoadBalancer, instance: ControllerInstanceId)(
```

Comment:
Can we name this method "instance"? (We should apply that to all Spis where applicable in a subsequent PR).